### PR TITLE
Support bridge confguration on segment

### PR DESCRIPTION
**Pull request by annakhm**
Add policy_bridge_profile data source and bridge_config clause
on segments.

Signed-off-by: Anna Khmelnitsky <akhmelnitsky@vmware.com>

### CODE REVIEW

File: nsxt/segment_common.go [hunk 1]
```diff
@@ -328,6 +350,12 @@ func getPolicyCommonSegmentSchema(vlanRequired bool, isFixed bool) map[string]*s
 			Default:      model.Segment_REPLICATION_MODE_MTEP,
 			ValidateFunc: validation.StringInSlice(replicationModeValues, false),
 		},
+		"bridge_config": {
+			Type:        schema.TypeList,
+			Description: "Beidge configuration",
```
Comment (ksamoray) [inline nsxt/segment_common.go:355]: Bridge?

**Review by ksamoray — CHANGES_REQUESTED** [review]

File: nsxt/segment_common.go (hunk 1 above)
Comment (annakhm) [reply nsxt/segment_common.go]: Thanks, fixed

**Review by 2ez4szliu — APPROVED** [review]

**Comment by ajgrinnell** [comment]
Thank you for working on this. I seem to be getting errors still even with the latest provider. I have pieced together what I can from the code and think I have it, but its still not working. If you have any ideas I would appreciate it. 

╷
│ Error: Unsupported block type
│ 
│   on segments.tf line 13, in resource "nsxt_policy_segment" "Overlay_Segment":
│   13:   bridge_config {
│ 
│ Blocks of type "bridge_config" are not expected here.
 terraform plan -var-file="secret.tfvars" 

##
│ Error: Invalid data source
│ 
│   on main.tf line 35, in data "nsxt_policy_bridge_profile" "TEST-Edge-Bridge":
│   35: data "nsxt_policy_bridge_profile" "TEST-Edge-Bridge" {
│ 
│ The provider vmware/nsxt does not support data source "nsxt_policy_bridge_profile".

##
 terraform version
Terraform v1.3.1
on linux_amd64
+ provider registry.terraform.io/vmware/nsxt v3.2.8

**Comment by annakhm** [comment]
Hi @ajgrinnell, sorry for late response, I've missed this comment. Bridge support was added with version 3.2.9

**Pull request merged**